recipes/pyahocorasick/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pyahocorasick" %}
+{% set version = "1.1.6" %}
+{% set sha256sum = "6bb71feccd515584edfb6923cd9e5c143da11cf0f7ffbc3c7d3eb2912cbd2026" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
please add `toolchain` since this compiles C-extensions.

Reply:
Added in 9b6887a